plugins/inputs/wireless/wireless_mac.go

```diff
@@ -0,0 +1,56 @@
+//go:build darwin
+// +build darwin
+
+package wireless
+
+import (
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+)
+
+// default executable path & flags
+const (
+	OSXCMD = "/System/Library/PrivateFrameworks/Apple80211.framework/Versions/Current/Resources/airport"
+	FLAG   = "-I"
+)
+
+func (w *Wireless) Gather(acc telegraf.Accumulator) error {
+	cmd := exec.Command(OSXCMD, FLAG)
+	wireless, err := internal.StdOutputTimeout(cmd, 2*time.Second) // cmd.Output()
```

Comment:
Please remove the comment at the end of line 24.

Reply:
Done. I'll submit the change with the rest once the above matter is decided. I'm hoping to only have to submit one further version before it can be merged.